otherlibs/stdune-unstable/spawn.ml

```diff
@@ -29,18 +7,6 @@ let spawn ?env ~prog ~argv ?(stdin = Unix.stdin) ?(stdout = Unix.stdout)
   let argv = Array.of_list argv in
   let env = Option.map ~f:Env.to_unix env in
   Pid.of_int
-    ( if Sys.win32 then
-      match env with
-      | None -> Unix.create_process prog argv stdin stdout stderr
-      | Some env -> Unix.create_process_env prog argv env stdin stdout stderr
-    else
-      match Unix.fork () with
-      | 0 -> (
-        try
-          ignore (Unix.sigprocmask SIG_SETMASK [] : int list);
```

Comment:
That's the bit we are loosing, and the reason why we inlined `Unix.create_process`.

Reply:
Good point. Well that's a shame. It might be worth investigating into writing better bindings for posix_spawn or using vfork in the future.

Reply:
We didn't use `posix_spawn` in `spawn` because it doesn't support changing the cwd BTW